cmd/up.go

```diff
@@ -994,6 +1077,8 @@ func createSSHCommand(
 	}
 	args = append(args, extraArgs...)
 
+	logger.Debug("Connecting with SSH command ", execPath, args)
```

Comment:
Do we still need this?

Reply:
I thought it would be a useful debug log so we have the flags the SSH connection made